iana/entid.go

```diff
@@ -0,0 +1,6 @@
+package iana
+
+type EntID uint32
+
+// EntIDCiscoSystems is IANA Enterprise ID for Cisco Systems
+const EntIDCiscoSystems EntID = 0x9
```

Comment:
Does it need to be hexa?

I would use decimal like in the IANA definitions:

https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers

Reply:
Yes, because the enterprise ID in https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/option_vivc.go#L14 is a uint32. 

Not sure if we should maintain consistency or convert it to uint32 where it is being used. Any thoughts?

Reply:
> Yes, because the enterprise ID s a uint32

What does that have to do with using decimal vs hexa numbers?

Reply:
Could go either way, if it makes more sense, I can have it as a decimal here and convert to uint32 where its being used

Reply:
With regards to the `uint32` conversion: The ideal way would be to make the `OptVendorClass` struct to use a `iana.EntID` type instead of uint32.

Similar to how it is done here:
https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/option_clientlinklayeraddress.go#L18

Reply:
You mean change VIVCIdentifier EntID  type from uint32 to iana.EntID right in https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/option_vivc.go#L14?

https://github.com/insomniacslk/dhcp/search?q=OptVendorClass does not seem to be related in this case.

Reply:
Can you use the decimal "9" instead of "0x9"? I think we would benefit from this in case more enterprise IDs are added.

Reply:
> You mean change VIVCIdentifier EntID type from uint32 to iana.EntID right in https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/option_vivc.go#L14?

Yes but it may be out of the scope of this PR, up to you.

> https://github.com/insomniacslk/dhcp/search?q=OptVendorClass does not seem to be related in this case.

That one should be changed as well but it is for v6, so yes, not related to this case.